sdk/communication/azure-communication-sms/pom.xml

```diff
@@ -56,6 +56,11 @@
       <artifactId>azure-communication-common</artifactId>
       <version>1.0.0-beta.3</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-communication-administration</artifactId>
```

Comment:
why does sms need to have dependency to admin? I am assuming only the tests need them, right?

Reply:
Yes, was thinking about the same thing. Only tests need them but we need to set up the phone numbers (in administration) before running the SMS tests so there is a dependency there. Any ideas on how we can avoid this?

Reply:
Not a Java expert here, maybe @srnagar has ideas?

Reply:
If a dependency is needed only for tests, you can set the `scope` of the dependency to `test`.

```xml
<dependency>
  <groupId>com.azure</groupId>
  <artifactId>azure-communication-administration</artifactId>
  <version>1.0.0-beta.3</version> <!-- {x-version-update;com.azure:azure-communication-administration;current} -->
  <scope>test</scope>
</dependency>
```